### PR TITLE
[api] Convert openSource to vd.open

### PR DESCRIPTION
**Pull request by anjakefala**

### CODE REVIEW

File: visidata/data.py [hunk 1]
```diff
@@ -9,9 +9,11 @@
 option('filetype', '', 'specify file type', replay=True)
 option('incr_base', 1.0, 'start value for column increments', replay=True)
 
+__all__ = ['SettableColumn', 'inputFilename', 'numrange', 'updateColNames', 'num', 'fillNullValues', 'inputPath', 'loadInternalSheet', 'open_txt']
+
 
 def _default_colnames():
-    'A B C .. Z AA AB .. ZZ AAA .. to infinity'
+    'a b c .. z aa ab .. zz aaa .. to infinity'
```
Comment (saulpw) [inline visidata/data.py]: why was this changed? The names are actually uppercase.

Comment (anjakefala) [reply visidata/data.py]: I have no idea, I noticed that in the diff, and assumed it was you. I should have asked, I will revert it!

File: visidata/data.py [hunk 2]
```diff
@@ -160,7 +162,7 @@ def openPath(vd, p, filetype=None):
     return openfunc(p)
 
 
-@VisiData.global_api
+@VisiData.api
```
Comment (saulpw) [inline visidata/data.py]: keep as `global_api` for now, we'll make that be the deprecation mechanism and clean it up later.  (this way other plugins/etc using it as a global should still work).
